### PR TITLE
Update amazon-s3-connector-reference.adoc

### PR DESCRIPTION
**Pull request by adam-lipka**

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 1]
```diff
@@ -398,134 +375,134 @@ Copies a source object to a new destination; to copy an object, the caller's acc
...
-| Destination User Metadata a| Object |  The new metadata of the destination object, that if specified, overrides that copied from the source object |  |
-| Modified Since a| DateTime |  The modified constraint that restricts this request to executing only if the object has been modified after the specified date. This constraint is specified but does not match, no copy is performed |  |
-| Unmodified Since a| DateTime |  The unmodified constraint that restricts this request to executing only if the object has not been modified after this date. This constraint is specified but does not match, no copy is performed |  |
-| Encryption a| String |  Encryption method for server-side encryption. Supported value AES256. |  |
-| Content Type a| String |  The destination object's Content-Type HTTP header |  |
-| Target Variable a| String |  The name of a variable that stores the output of an operation. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the targer variable. |  `#[payload]` |
+** GLACIER |  +++one of StorageClass enumerated values, defaults to StorageClass#STANDARD+++ |  +++STANDARD+++ |
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Is storageClass#STANDARD correct in the published doc (View File view)?

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 2]
```diff
@@ -684,2512 +658,2530 @@ this operation creates a zero-byte object.
...
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Bucket Name a| String |  The bucket whose cors to retrieve. |  | x
-| Target Variable a| String |  The name of a variable that stores the output of an operation. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the targer variable. |  `#[payload]` |
+| Bucket Name a| String |  +++The bucket whose cors to retrieve.+++ |  | x
+| Target Variable a| String |  +++The name of a variable to store the operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ |  `#[payload]` | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A r
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: The description for Type does not appear like it's rendering correctly in the View File view. Please check this.

Comment (adam-lipka) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: This is what I see in View File view. Could you please advise me on what is incorrect here?
image

If I click on `Reconnect` or `Reconnect Forever` links it shows me description of these with correct comments.

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 1 above)
Comment (adam-lipka) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: It should show just `STANDARD`, probably some bug in markdown generation from javadocs. However, we should not modify it in code as it's correct in Javadoc. We can modify it by hand here but we will need to do it after every release.

Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Please modify this by hand.

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 2 above)
Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: The issue is in the Output section for the Get Bucket Versioning Configuration.
Screen Shot 2019-06-17 at 9 18 41 AM

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 1 above)
Comment (adam-lipka) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Fixed.

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 2 above)
Comment (adam-lipka) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Thanks! It should be list of OFF, ENABLED and SUSPENDED values but I couldn't make it look like that. I updated it and now it appears syntactically correct to me but it still renders wrong. Could you please take a look?

**Review by dhirsch1 — APPROVED** [review]

**Review by dhirsch1 — APPROVED** [review]
All set

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**